### DISCUSSION TITLE
Function for GLFW_AUTO_ICONIFY / Minimizing app when fullscreened and focus is lost

### DISCUSSION THREAD

**Discussion by SolarLune** [hajimehoshi/ebiten#1729] [Ideas]
Hello!

Currently on desktop, if you fullscreen an Ebiten app and then click away from it elsewhere (on another monitor, for example), the app minimizes. It'd be nice to add a function to allow the user to disable this functionality. Looks like this should be able to be accomplished with [GLFW_AUTO_ICONIFY](https://www.glfw.org/docs/3.3/window_guide.html#GLFW_AUTO_ICONIFY_attrib). Does this seem like a good idea?

**Comment by hajimehoshi** [comment]
Hi,

Actually Ebiten prevented the app from being iconified, but we found an issue https://github.com/hajimehoshi/ebiten/issues/1504 then I've reverted the change.

I've filed https://github.com/hajimehoshi/ebiten/issues/1506 to iconify the window only when appropriated.

**Reply by SolarLune** [reply]
Hello, late to reply - sorry about that! I see, so it's not really a straightforward option, I understand. Thanks for responding!
